flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/runner/DispatcherLeaderProcessImplTest.java

```diff
@@ -239,6 +246,109 @@ public void closeAsync_duringJobRecovery_preventsDispatcherServiceCreation() thr
 		}
 	}
 
+	@Test
+	public void onRemovedJobGraph_cancelsRunningJob() throws Exception {
```

Comment:
IIRC method name always follows lowerCamelCase?

Reply:
In this test class I'm trying to follow a new naming scheme which I believe is more helpful and is described here: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html. The idea is to have more descriptive names for tests.

Reply:
interesting to try it out :-)